tests/e2e/playground/specs/playground.spec.js

```diff
@@ -0,0 +1,17 @@
+// Run via a command like the f./support/helpers
+// VUE_APP_API_URL=http://localhost:1337 ./node_modules/.bin/cypress run --spec tests/e2e/specs/playground.js
+
+import { setupGameAsP0 } from '../../support/helpers';
+
+const attempts = 400;
```

Comment:
This allows us to run a single test N times. I've been using this to try and track down that `Forbidden` error without much luck yet. I can reproduce but can't fix it. No reason to hold up this PR imo.